common/scala/src/main/resources/application.conf

```diff
@@ -108,3 +108,15 @@ whisk {
         std = 256 m
     }
 }
+
+//whisk.logstore {
+//    splunk {
+//        host = "splunk-host"
+//        port = 8080
+//        user = "splunk-user"
+//        password = "splunk-pass"
+//        index = "splunk-index"
+//        log-message-field = "log_message"
+//        activation-id-field = "activation_id"
+//    }
+//}
```

Comment:
Why commented out? We should start building a `reference.conf` with sensible defaults. As long as the splunk logstore is not loaded, this will not take effect anyway?

Reply:
just for sample config, will remove till there is a better place to locate this